scripts/swupd-completion.sh

```diff
@@ -0,0 +1,139 @@
+#!/bin/bash
+
+# This script creates a _swupd function to be used by
+# completion built-in bash command to complete swup
+# It reads options and subcommands from swup --help
+# Also it reads options for each subcommand recursively
+
+
+SCRIPTNAME="swupd.bash"
+COMPLETIONFUNCTION="_swupd"
+SWUPDCOMMAND="./swupd"
+
+optionsfromhelp=""
+subcommandsfromhelp=()
+optionsfromsubcommand=()
+
+#Checks if swupd commad exists in current shell
+if [ -a $SWUPDCOMMAND ] ; then
```

Comment:
I wasn't aware of the -a test option :-)  Since -e is more commonly used, can you use it instead?